src/inventory_client/inventory_client.go

```diff
@@ -82,8 +87,13 @@ func (c *inventoryClient) GetEnabledHostsNamesIds() (map[string]string, error) {
 	if err != nil {
 		return nil, err
 	}
-	for hostId, hwInfo := range hosts {
-		namesIdsMap[hwInfo.Hostname] = hostId
+	for hostId, hostData := range hosts {
+		//hostname := hostData.host.RequestedHostname
```

Comment:
remove it?

Reply:
yes, i will do it once i will understand how and when we update the go.mod